### PR TITLE
47206_static_pages

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,56 +1,10 @@
...
-  #     resources :products
-  #   end
+	scope "(:locale)", :locale => /en/ do
+
+	root 							 'static_pages#home'
+	get 'home' 			=> 'static_pages#home'
+	get 'about' 		=> 'static_pages#about'
+	get 'contact'		=> 'static_pages#contact'
+	
+	end
```
Comment (mdmahmudurrahman) [inline config/routes.rb]: OK. Right now.

File: Gemfile [hunk 2]
```diff
@@ -6,6 +6,7 @@ gem 'rails', '4.2.5'
 # Use sqlite3 as the database for Active Record
 #gem 'sqlite3'
 gem 'mysql2'
+gem 'bootstrap-sass',				'3.2.0.0'
```
Comment (shakil-fg) [inline Gemfile]: spacing

File: app/assets/javascripts/static_pages.coffee [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (shakil-fg) [inline app/assets/javascripts/static_pages.coffee]: remove empty file

File: app/assets/stylesheets/static_pages.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (shakil-fg) [inline app/assets/stylesheets/static_pages.scss]: remove file

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -2,4 +2,15 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :set_locale
+  
+
```
Comment (shakil-fg) [inline app/controllers/application_controller.rb]: remove additional spacing

File: app/helpers/static_pages_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (shakil-fg) [inline app/helpers/static_pages_helper.rb]: remove file

File: config/locales/en.yml [hunk 7]
```diff
@@ -21,3 +21,12 @@
```
Comment (shakil-fg) [inline config/locales/en.yml]: empty line in the beginning

**Comment by shakil-fg** [comment]
OK

**Comment by mdmahmudurrahman** [comment]
ok

File: app/controllers/application_controller.rb [hunk 8]
```diff
@@ -2,4 +2,14 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :set_locale
+  
+  def set_locale
+  		I18n.locale = params[:locale] || I18n.default_locale
```
Comment (chuanhtuan) [inline app/controllers/application_controller.rb]: indent

File: app/views/layouts/_header.html.erb [hunk 9]
```diff
@@ -0,0 +1,10 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "Framgia Training System", '#', id: "logo" %>
```
Comment (chuanhtuan) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/static_pages/home.html.erb [hunk 10]
```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (chuanhtuan) [inline app/views/static_pages/home.html.erb]: ???
really, how about layout :(

**Comment by chuanhtuan** [comment]
remove .idea folder

File: app/views/layouts/application.html.erb [hunk 11]
```diff
@@ -5,10 +5,49 @@
...
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+  <meta name="description" content="">
+  <meta name="author" content="">
+  <link rel="icon" href="../../favicon.ico">
+
+  <!-- Bootstrap core CSS -->
+  <link href="../../dist/css/bootstrap.min.css" rel="stylesheet">
```
Comment (chuanhtuan) [inline app/views/layouts/application.html.erb]: remove
use "require tree ."

File: app/views/layouts/application.html.erb [hunk 12]
```diff
@@ -5,10 +5,49 @@
...
+
+  <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+  <link href="../../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
```
Comment (chuanhtuan) [inline app/views/layouts/application.html.erb]: remove
use "require tree ."

File: app/views/layouts/application.html.erb [hunk 13]
```diff
@@ -5,10 +5,49 @@
...
 
-<%= yield %>
+  <!-- Custom styles for this template -->
+  <link href="jumbotron.css" rel="stylesheet">
```
Comment (chuanhtuan) [inline app/views/layouts/application.html.erb]: remove
use "require tree ."

File: app/views/layouts/application.html.erb [hunk 14]
```diff
@@ -5,10 +5,49 @@
...
 
-</body>
+  <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
+  <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
+  <script src="../../assets/js/ie-emulation-modes-warning.js"></script>
```
Comment (chuanhtuan) [inline app/views/layouts/application.html.erb]: remove
use "require tree ."

File: config/routes.rb [hunk 15]
```diff
@@ -1,56 +1,10 @@
...
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
+	scope "(:locale)", :locale => /en/ do
+
+	root 							 'static_pages#home'
```
Comment (chuanhtuan) [inline config/routes.rb]: too many space

File: app/controllers/application_controller.rb [hunk 16]
```diff
@@ -2,4 +2,14 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :set_locale
+  
+  def set_locale
+		I18n.locale = params[:locale] || I18n.default_locale
```
Comment (chuanhtuan) [inline app/controllers/application_controller.rb]: indent

File: config/routes.rb [hunk 17]
```diff
@@ -1,56 +1,8 @@
...
-  #   resources :posts, concerns: :toggleable
-  #   resources :photos, concerns: :toggleable
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
+	scope "(:locale)", :locale => /en/ do
```
Comment (chuanhtuan) [inline config/routes.rb]: indent 2

File: config/routes.rb [hunk 18]
```diff
@@ -1,56 +1,8 @@
...
+		root	'static_pages#home'
```
Comment (chuanhtuan) [inline config/routes.rb]: indent 2

File: app/controllers/application_controller.rb (hunk 16 above)
Comment (noor-framgia) [inline app/controllers/application_controller.rb]: sir, really sorry for these inconveniences, but actually i have only indent 2 in my local files, but when git shows big indents here. i dont know why this is happening.

**Pull request merged**